sdk/cognitivelanguage/azure-ai-language-conversations/samples/reslotion_and_pii/sample_conv_pii_transcript_input_dict_parms.py

```diff
@@ -0,0 +1,140 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_conv_pii_transcript_input_dict_parms.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze a conversation for PII (personally identifiable information).
+
+    For more info about how to setup a CLU conversation project, see the README.
+
+USAGE:
+    python sample_conv_pii_transcript_input_dict_parms.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_CONVERSATIONS_ENDPOINT                       - endpoint for your CLU resource.
+    2) AZURE_CONVERSATIONS_KEY                            - API key for your CLU resource.
+"""
+
+from azure.ai.language import conversations
+
+
+def sample_conv_pii_transcript_input_dict_parms():
+    # [START analyze_conversation_app]
+    # import libraries
+    import os
+    from azure.core.credentials import AzureKeyCredential
+
+    from azure.ai.language.conversations import ConversationAnalysisClient
+
+    # get secrets
+    endpoint = os.environ["AZURE_CONVERSATIONS_ENDPOINT"]
+    key = os.environ["AZURE_CONVERSATIONS_KEY"]
+
+    # analyze quey
+    client = ConversationAnalysisClient(endpoint, AzureKeyCredential(key))
+    with client:
+
+        poller = client.begin_submit_conversation_job(
+            body={
+                "displayName": "Analyze conversations from xxx",
```

Comment:
Maybe something like "Analyze PII in conversation"?

Reply:
i don't agree with this
simply because in the async api, we can submit either `issue resolution` or `conv pii` tasks
but weirdly enough, autorest appends `begin` in front of the async operation after rename
do you have an idea how to fix that?